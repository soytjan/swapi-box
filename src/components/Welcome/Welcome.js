import React from 'react';
import PropTypes from 'prop-types';
import ScrollContainer from '../ScrollContainer/ScrollContainer';
import './Welcome.css';

const Welcome = ({film}) => {
  return (
    <div className='welcome'>
      I'm Welcome
      <ScrollContainer {...film} />
    </div>
  )
}

Welcome.propTypes = {
  onClick: PropTypes.func,
};

Welcome.propTypes = {
  film: PropTypes.object,
};

export default Welcome;