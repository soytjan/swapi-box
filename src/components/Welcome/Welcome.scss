.welcome {
  width: 100%;
  height: 100%;
  overflow: hidden;
  background: #000;
  padding: 0 20%;
}