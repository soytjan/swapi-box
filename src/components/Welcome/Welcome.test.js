import React from 'react';
import Welcome from './Welcome';

describe('Welcome', () => {
  it('expect to exist', () => {
    expect(Welcome).toBeDefined();
  })
})