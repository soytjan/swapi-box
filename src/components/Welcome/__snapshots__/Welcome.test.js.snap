// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Welcome
    film={
        Object {
            "crawl": "War! The Republic is crumbling under attacks by t…ate mission to rescue the captive Chancellor....",
            "releaseDate": "2005-05-19",
            "title": "Revenge of the Sith",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ScrollContainer
        crawl="War! The Republic is crumbling under attacks by t…ate mission to rescue the captive Chancellor...."
        releaseDate="2005-05-19"
        title="Revenge of the Sith"
/>,
      "className": "welcome",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "crawl": "War! The Republic is crumbling under attacks by t…ate mission to rescue the captive Chancellor....",
        "releaseDate": "2005-05-19",
        "title": "Revenge of the Sith",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ScrollContainer
          crawl="War! The Republic is crumbling under attacks by t…ate mission to rescue the captive Chancellor...."
          releaseDate="2005-05-19"
          title="Revenge of the Sith"
/>,
        "className": "welcome",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "crawl": "War! The Republic is crumbling under attacks by t…ate mission to rescue the captive Chancellor....",
          "releaseDate": "2005-05-19",
          "title": "Revenge of the Sith",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
